Source/Wit/Private/Wit/TTS/WitTtsService.cpp

```diff
@@ -423,7 +423,7 @@ void UWitTtsService::OnVoicesRequestComplete(const TArray<uint8>& BinaryResponse
  * @param ErrorMessage [in] the error message
  * @param HumanReadableErrorMessage [in] longer human readable error message
  */
-void UWitTtsService::OnVoicesRequestError(const FString ErrorMessage, const FString HumanReadableErrorMessage)
+void UWitTtsService::OnVoicesRequestError(const FString& ErrorMessage, const FString& HumanReadableErrorMessage)
```

Comment:
what is the reason to add "&"?  for memory efficiency?

Reply:
Yeah, passing by value instead of reference causes unnecessary copies of data.